docs/data-ai/ai/run-inference.md

```diff
@@ -4,6 +4,100 @@ title: "Run inference on a model"
 weight: 50
 layout: "docs"
 type: "docs"
-no_list: true
-description: "TODO"
+modulescript: true
+description: "Run inference on a model with a vision service or an SDK."
 ---
+
+After deploying an ml model, you need to configure an additional service to use the inferences the deployed model makes.
+You can run inference on an ML model with a vision service in the app or use an SDK.
+
+## Use a vision service
+
+Vision services work to provide computer vision.
+They use an ML model and apply it to the stream of images from your camera.
+
+{{<resources_svc api="rdk:service:vision" type="vision">}}
+
+{{< readfile "/static/include/create-your-own-mr.md" >}}
+
+Note that many of these services have built in ML models, and thus do not need to be run alongside an ML model service.
+
+One vision service you can use to run inference on a camera stream if you have an ML model service configured is the `mlmodel` service.
+
+### Configure an mlmodel vision service
+
+Add the `vision / ML model` service to your machine.
+Then, from the **Select model** dropdown, select the name of the ML model service you configured when [deploying](/data-ai/ai/deploy/) your model (for example, `mlmodel-1`).
+
+**Save** your changes.
+
+### Test your changes
```

Comment:
```suggestion
### Check vision service 
```

Reply:
sorry-- why not have this info on here? is it too self-explanatory?